sql/_27_banana_qa/issue_12072_with_fullscan/answers/_002_multi_tables.answer

```diff
@@ -40,7 +40,7 @@ dba.t1     1     idx     1     a     A     1     null     null     YES     BTREE
 0
 ===================================================
 Table    Non_unique    Key_name    Seq_in_index    Column_name    Collation    Cardinality    Sub_part    Packed    Null    Index_type    Func    Comment    Visible    
-dba.t2     1     idx     1     a     A     1     null     null     YES     BTREE     null     null     YES     
+dba.t2     1     idx     1     a     A     4     null     null     YES     BTREE     null     null     YES     
```

Comment:
Why did the results change?

Reply:
'update statistics' statement updates all statistics information in the specified table. So cardinality is updated as containing all the inserted data. Therefor revising answer is necessarry.

Reply:
OK.